kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Utils.java

```diff
@@ -177,49 +177,6 @@ public static void waitUntilReadyOrFail(Future<?> future, long amount, TimeUnit
     }
   }
 
-  /**
-   * Closes the specified {@link ExecutorService}.
-   * @param executorService   The executorService.
-   * @return True if shutdown is complete.
-   */
-  public static boolean shutdownExecutorService(ExecutorService executorService) {
```

Comment:
The idea behind this utility was to give time to the executor service to cleanly shutdown.
By cleanly shutdown I mean to try and finish running jobs before actually forcing termination.

TBH, I don't recall if a a more drastic solution (like the suggested `shutdownNow()`) will have side-effects, but I would like to know what was the motivation behind the particular change.

Reply:
I can understand the motivation for the utility, but it's application in practice meant that prior to calling you needed to cancel any long running jobs - otherwise you'd hit a potentially unnecessary wait.  If the calling scenario cancels running things anyway, then you might as well just call shutdownNow - which matches the remaining places the utility method was used.  My inclination is to get rid of the utility method to avoid future confusion, but it could be left with a caveat to just call shutdownNow directly if you okay with cancelling running jobs.

Reply:
Given, that I haven't got a use-case where waiting for completion would be required, I'll accept that.